packages/slate/src/models/leaf.js

```diff
@@ -189,3 +350,9 @@ Leaf.prototype[MODEL_TYPES.LEAF] = true
  */
 
 export default Leaf
+
+/*
+ * An pre-created Leaf to speed up empty Leaf creation in some cases
+ */
+
+export const EMPTY_LEAF = Leaf.create({})
```

Comment:
Can you omit this an add it in a new pull request later? I don't want to add unnecessary things here. This should be proven by a separate pull request that it actually impacts the benchmarks.